src/sql/workbench/services/notebook/browser/models/cellEdit.ts

```diff
@@ -47,32 +53,38 @@ export class DeleteCellEdit implements IResourceUndoRedoElement {
 	type: UndoRedoElementType.Resource = UndoRedoElementType.Resource;
 	label: string = localize('deleteCellEdit', "Delete Cell");
 	resource = this.model.notebookUri;
+	cellOperation = { cell_operation: this.label };
 
 	constructor(private model: NotebookModel, private cell: ICellModel, private index: number) {
 	}
 
 	undo(): void {
 		this.model.insertCell(this.cell, this.index, false);
+		this.model.createNotebookTelemetryEvent(TelemetryKeys.NbTelemetryAction.UndoCell, this.cellOperation);
 	}
 
 	redo(): void {
 		this.model.deleteCell(this.cell, false);
+		this.model.createNotebookTelemetryEvent(TelemetryKeys.NbTelemetryAction.RedoCell, this.cellOperation);
 	}
 }
 
 export class AddCellEdit implements IResourceUndoRedoElement {
 	type: UndoRedoElementType.Resource = UndoRedoElementType.Resource;
 	label: string = localize('addCellEdit', "Add Cell");
 	resource = this.model.notebookUri;
+	cellOperation = { cell_operation: this.label };
```

Comment:
We shouldn't send localized strings in telemetry. Use a static ID for each instead.